chrono-tz-build/Cargo.toml

```diff
@@ -19,6 +19,6 @@ regex = ["dep:regex"]
 [dependencies]
 parse-zoneinfo = { version = "0.3" }
 regex = { default-features = false, version = "1", optional = true }
-phf = { version = "0.11", default-features = false }
+phf = { version = "0.11", default-features = false, optional = true }
```

Comment:
Can you add a comment here that we only have this dependency in order to forward the `uncased` feature to `phf_shared`?

Reply:
Actually replaced `phf` with `phf_shared` and added comment.